File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F17.P1.java
import java.io.FileInputStream;
import java.io.IOException;

class Main {
    // BEGIN PROBE F17.P1
    static int readAll() throws IOException {
        try (var in = new FileInputStream("data.txt")) {
            String text = new String(in.readAllBytes());
            return text.getBytes().length;
        }
    }
    // END PROBE F17.P1

    public static void main(String[] args) throws IOException {
        System.out.println(readAll());
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/kotlin/F17.P1.kt
import java.io.File

// BEGIN PROBE F17.P1
fun readAll(): Int {
    val text = File("data.txt").bufferedReader().use { it.readText() }
    return text.toByteArray().size
}
// END PROBE F17.P1

fun main() {
    println(readAll())
}
